### ISSUE TITLE
Windows failure for npm start

### ISSUE THREAD

**Issue by johnymontana** [grand-stack/grand-stack-starter#77]
From Vincent:

```
For anyone on Windows: NPM run start will fail due to " DEBUG=neo4j-graphql-js " in /api/package.json
Use something like: "start:dev": "SET DEBUG=neo4j-graphql-js && nodemon --watch src --ext js,graphql --exec babel-node  src/index.js"
```

**Comment by johnymontana** [comment]
Look at fixing this with https://www.npmjs.com/package/cross-env

**Comment by johnymontana** [comment]
Fixed in a3d5e3b53cf98a7b9d03b9fc35a901b5c94ae2c9
